src/graph/validator/SequentialValidator.cpp

```diff
@@ -24,6 +25,11 @@ Status SequentialValidator::validateImpl() {
         "given.",
         static_cast<int64_t>(sentence_->kind()));
   }
+  size_t querySize = sentence_->toString().size();
```

Comment:
I think you could check origin query string.

Reply:
done